xExtension-YouTube/extension.php

```diff
@@ -106,7 +106,7 @@ public function embedYouTubeVideo($entry)
     {
         $link = $entry->link();
 
-        if (stripos($link, 'www.youtube.com/watch?v=') === false) {
+        if (stripos($link, 'www.youtube.com/watch?v=') === false and stripos($link, 'videos/watch/') === false) {
```

Comment:
It could be good to have a slightly stricter filter, to avoid too many false detections.
Maybe a regular expression such as `preg_match('#^https?://www.youtube.com/watch?v=|/videos/watch/[0-9a-f-]{10}$#', ...)`

Reply:
```if (preg_match('#^https?://www\.youtube\.com/watch\?v=|/videos/watch/[0-9a-f-]{36}$#', $link)) {```

Reply:
I haven‘t checked peertube documentation, but using videos/watch/ as an identifier might result in a lot of false positives.

I still want to enhance the YouTube plugin to be a more versatile multi-Video-platform Plugin with options to activate single sites.

Reply:
I had the same concert, thus the regex suggestion

Reply:
Yep I guess there are no other ways to recognize it at least from url, and in the xml there is no mandatory mention of peertube